ETC.tsv

```diff
@@ -3305,7 +3305,7 @@ ETC_20150317_003304	] should be completed to pass through
 ETC_20150317_003305	The song seems to touch the heart so let's copy it
 ETC_20150317_003306	Found the lighted fountain
 ETC_20150317_003307	Putting up
-ETC_20150317_003308	You cannot pass because the monster is blocking the way. {nl}
+ETC_20150317_003308	You cannot pass because a monster is blocking the way. {nl}
```

Comment:
The original line is fine. They could be referring to specific monster not just any monster.